validator/gke_enable_shielded_nodes_test.rego

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GCPGKEEnableShieldedNodesConstraintV1
+
+import data.validator.gcp.lib as lib
+
+all_violations[violation] {
```

Comment:
Do you mind updating the tests to use the new util we just added (should make them simpler)? #289

Reply:
Nice predicate. Much easier to write tests now!